src/input_cp2k_properties_dft.F

```diff
@@ -1603,6 +1603,31 @@ SUBROUTINE create_tddfpt2_section(section)
       CALL section_add_subsection(section, subsection)
       CALL section_release(subsection)
 
+      ! SOC functional
+
+      CALL section_create(subsection, __LOCATION__, name="SOC", &
+                          description="Is jet to be implemented", &
+                          n_keywords=2, n_subsections=0, repeats=.FALSE.)
+
+      CALL keyword_create(keyword, __LOCATION__, name="SINGLETT", &
+         description="Number of singlett states", &
```

Comment:
I think `singlet` and `triplet` is the correct spelling.

Reply:
Thank you very much, you are absolutely right.  This error should be resolved now.